src/platform/utilities/environment/index.js

```diff
@@ -1,43 +1,85 @@
-const _Environments = {
-  preview: {
-    API_URL: 'https://api.va.gov',
-    BASE_URL: 'https://preview.va.gov',
-  },
-  vagovprod: {
-    API_URL: 'https://api.va.gov',
+/*
+ * Configuration for the current environment.
+ * @module platform/utilities/environment
+ */
+
+import ENVIRONMENTS from '../../../site/constants/environments';
+
+// eslint-disable-next-line no-undef
+const BUILDTYPE = __BUILDTYPE__;
+
+const {
+  WEB_PORT: RESERVED_E2E_PORT = 3333,
+  API_PORT: E2E_API_PORT = 3000,
+} = process.env;
```

Comment:
It took me a while to understand what these environment variables were used for. I think there may have been some confusion at some point about how environment variables are made available throughout Webpack-compiled code, versus being passed into Mocha/E2E/other Node applications. I'd like to address the env variables/port numbers sometime down the road, since `3333` is not just an arbitrary number, and is instead a contract with the Nightwatch config to indicate that we're running E2E tests.